js/functional/palindrome.js

```diff
@@ -27,8 +48,18 @@ function isPalindromeUsingLoop(value) {
  * @param {number} index
  */
 function isPalindromeUsingRecursive(value, index = 0) {
-  // write your code here
-  return true;
+  value = parseString(value);
+
+  const cleanedValue = cleanString(value);
+  if (index >= Math.floor(cleanedValue.length / 2)) {
```

Comment:
Put an extra line break at this "if" statement to make it easier to read by the other.